README.md

```diff
@@ -1,2 +1,111 @@
 # analytical-engineer-challenge
 🚀 The Pleo Analytical Engineer Challenge
+
+
+## Stewards: @team-poseidon
+https://linear.app/pleo/project/stewards-analytical-engineer-challenge-192436dac756
+
+## Welcome!
+This repository houses the Pleo Analytical Engineer challenge. It contains a mix of detective work, ops-like work, programming and big-picture perspective. Your answer to the challenge will help us get to know you, and provide a fun context for subsequent discussion in our interviews :)
+
+We hope you enjoy it!
+
+## Introduction
+We have created an imaginary scenario which we hope will allow you to show off your analytical engineering skills while also having some fun:
+
+You've been tasked with taking over a small prototype for an ETL (Extract - Transform - Load) pipeline, meant to collate some data from several sources into a Data Product. The original author never finished the prototype, and they unfortunately also left very little documentation on how the prototype works! Part of this challenge is detective work, and so we've been intentionally vague in terms of describing how it works.
+
+Luckily you're available to help investigate this, determine what needs doing, and fix it so that it becomes a functional prototype! 
+
+## How to submit a solution
+
+When you are done with the below challenge, you have 2 choices:
+
+1. You can set it up as a private GitHub repository and invite the reviewers to the repository.
+1. You can zip the solution up and send it by e-mail to the reviewers.
```

Comment:
can we maybe skip this step? otherwise we're going to ask anyway for the access on the github repository because we're curious on the commits being made. I've never once looked at the zip archive

Reply:
and I think if we ever find someone who will explain he doesn't have an account on github ..